src/app/components/infinite-scroll/index.html

```diff
@@ -0,0 +1,29 @@
+<sky-demo-page
+  pageTitle="Infinite scroll">
+
+  <sky-demo-page-summary>
+    The infinite scroll component allows users to load data as the page scrolls.
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="hasMore"
+      defaultValue="false"
+      isOptional="true"
+    >
+    A boolean specifying whether there is more data to load. If this is false, infinite scroll will hide and stop calling the load function.
+    </sky-demo-page-property>
+
+    <sky-demo-page-property
```

Comment:
Insert an opening "sky-demo-page-properties" here with "sectionHeading="Events" " to create a separate section with an Events header.